frontend/src/components/identities/AccountCreationForm.vue

```diff
@@ -11,7 +11,7 @@
   @create="submit"
   >
   <template v-slot:form.role="{ step }">
-    <account-role-form :ref="`form_${step}`" v-model="account.role" :account="account" />
+    <account-role-form :ref="`form_${step}`" v-model="account.role" :account="account" :authUserRole="authUser.role" />
```

Comment:
Since there is only one authUser and it comes from a store, maybe you could use the getter directly inside the account-role-form component?